### PR TITLE
KH edits on non-integral calculations spec

### PR DESCRIPTION
**Pull request by mgudemann**
see #837 
split PR into separate parts

### CODE REVIEW

File: shelley/chain-and-ledger/dependencies/non-integer/doc/iohk.sty [hunk 1]
```diff
@@ -36,9 +36,9 @@
 \newenvironment{question}
   {\begin{bclogo}[logo=\bcquestion, couleur=orange!10, arrondi=0.2]{ QUESTION}}
   {\end{bclogo}}
-\newenvironment{todo}
-  {\begin{bclogo}[logo=\bcoutil, couleur=red!5, couleurBarre=red, arrondi=0.2]{ TODO}}
-    {\end{bclogo}}
+% \newenvironment{todo}
+%   {\begin{bclogo}[logo=\bcoutil, couleur=red!5, couleurBarre=red, arrondi=0.2]{ TODO}}
+%     {\end{bclogo}}
```
Comment (JaredCorduan) [inline shelley/chain-and-ledger/dependencies/non-integer/doc/iohk.sty]: any reason to keep the commented environment?

**Comment by kevinhammond** [comment]
Only that it was in the original style file.

File: shelley/chain-and-ledger/dependencies/non-integer/doc/iohk.sty (hunk 1 above)
Comment (kevinhammond) [reply shelley/chain-and-ledger/dependencies/non-integer/doc/iohk.sty]: Unfortunately the environment conflicts with the todonote package...

**Comment by kevinhammond** [comment]
There are a copy of issues for @mgudemann to look at, but these should be done in the next version.  We should freeze and release this one.

What is non-integral exponentiation used for?

The figures could be improved by e.g. adding labels.

**Comment by kevinhammond** [comment]
The stack build seems to be irrelevant to the document?

**Comment by mgudemann** [comment]
@kevinhammond  plese create an issue for changing the title page. I think we should get sign off on changing this. While you marked it with restricted dissemination, we currently publish the latest `master` version via the  Hydra build. These two things do not fit together.

**Pull request merged**